### DISCUSSION TITLE
Configurazione valori +/-

### DISCUSSION THREAD

**Discussion by scuolagioco** [iisgiua/giuaschool#335] [Richieste di aiuto] [answered]
Salve,
come posso cambiare i valori del + e del - dai templates?

**Comment by trinko** [comment, marked as answer]
Attualmente non è una cosa facilmente personalizzabile. 
Devi conoscere il javascript e andare a modificare la funzione "formatter" presente nel template, che gestisce l'inserimento corretto del valore. E non solo quello.
